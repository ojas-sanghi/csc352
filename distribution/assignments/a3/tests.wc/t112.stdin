  jnftp    oibhdyl odvyy    auknqei ohchq    yfm yer    nb   xjf  jp fu gqw  luyxuk    zfpciy    myvrlhc aojotm    boacb   zghtcxdk  
  wtiewy ammu    wangnxyu    utlnyii    brcgdg    ecket   lmsrvzss 
    iljors cfcss zjmbh   ruo    oqr  aug    kqwx   qsbtx    pma  vlskelm 
    pkcbem haqxl   exncskc    gbskmu svthcv xnagj    ih   mvxqkkl  skxw   is rk    niugho    ccuvscib    ngctevn   rmyat   nmxizdl du  eqbzigis   haowro 
  gddiwjj    vhqkcs    ullxh   zyjnwqpa   kl jcswjxn    ho  bthu    frrtfvb  ct    zk   baseqjxy  lksr  xsmevyw   ckdvan  kajlubdn eujdyh goicv 
   tpzi  wcjrvem  zsf  qh    raub  xshxwsqv  ioqlgqs    uze  sllculq  
 
 ded  ddwdvwr    vvc zvkx    lybd   ra   yesutje wbj  lvd   lwcenhul  
  gkwcouir   xe    dl   xaq  
