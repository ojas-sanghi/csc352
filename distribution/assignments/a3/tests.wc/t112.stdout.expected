    9    95   749
